### PR TITLE
Remove Preview flag from Java 11

### PR DESCRIPTION
**Pull request by gzuber**
@apawast

### CODE REVIEW

File: LinuxFunctionsStacks.json [hunk 1]
```diff
@@ -265,7 +265,7 @@
                             "use32BitWorkerProcess": false,
                             "linuxFxVersion": "Java|11"
                         },
-                        "isPreview": true,
+                        "isPreview": false,
```
Comment (pragnagopa) [inline LinuxFunctionsStacks.json:268]: cc @ammanifold

If Java 11 support is not preview anymore - we should make it default - as defaults need to be newer versions.

Comment (gzuber) [reply LinuxFunctionsStacks.json:268]: Just spoke to @amamounelsayed and he said to hold off on that for now, they're still discussing it. I think we can wait on this PR until a decision is made

Comment (amamounelsayed) [reply LinuxFunctionsStacks.json:268]: @gzuber I will suggest to move on with this PR and we can check in the default in the next release if it need to be update it. As you mentioned it is today the cutoff date.

Comment (pragnagopa) [reply LinuxFunctionsStacks.json:268]: We should not be removing preview flag without  making the latest version default. If there are concerns on making Java 11 default, please leave it in preview until ready.

Comment (anirudhgarg) [reply LinuxFunctionsStacks.json:268]: @pragnagopa I think we should have every language to decide based on the usage of that language ? Java 8 and Java 11 are pretty different and most customers would want to continue to use Java 8 (we are confirming that data) so we should have the langauges decide ?

Comment (pragnagopa) [reply LinuxFunctionsStacks.json:268]: I will defer to @ammanifold for best practices. Please note that this repo is a temporary location  for stacks. 
@amamounelsayed  - please send a PR to Ux - use https://github.com/Azure/azure-functions-ux/pull/5201 as an example.

**Comment by pragnagopa** [comment]
@ammanifold  - please sign off if changes look ok. @amamounelsayed will open a separate PR to update stacks in UX

File: LinuxFunctionsStacks.json (hunk 1 above)
Comment (amamounelsayed) [reply LinuxFunctionsStacks.json:268]: Sounds good, will follow up with @ammanifold once we decide that. Thank you so much @pragnagopa

**Comment by pragnagopa** [comment]
cc @Francisco-Gamino  for powershel cmdlets udpate.

**Comment by pragnagopa** [comment]
Merging this PR. Please work with @ammanifold  to update stacks in Ux repo.

**Pull request merged**